assemblies/assembly-planner-configuration.adoc

```diff
@@ -378,6 +369,7 @@ In Eclipse, `debug` logging to the console tends to cause congestion with score
 Set the logging level to `debug` logging to see when the phases end and how fast steps are taken.
 
 The following example shows output from debug logging:
+COMMENT: Anna, we problem need to copy real log data.
```

Comment:
[trace.log](https://github.com/emmurphy1/kie-docs/files/6960661/trace.log)

Here is an example of the full log with debug logging